aten/src/ATen/PTThreadPool.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
```

Comment:
I wonder if at some point we want to move this into core.

Reply:
shouldn't an issue to move this, as I understand..

Reply:
oh, if you mean c10/core then c10 cannot depend on ATen, so that's why I had to move to ATen
soon it's not going to an issue as we're merging the libraries